src/components/RegistrationForm/RegistrationForm.tsx

```diff
@@ -117,7 +117,7 @@ const RegistrationForm: React.FC<Props> = ({
         <Checkbox
           key={index}
           name={consent.name}
-          value={consent.value || ''}
+          value={''}
```

Comment:
Mistake I think?

Reply:
I cannot explain how this slipped through, is indeed a mistake and was solely for testing purposes! Thanks 😄